### DISCUSSION TITLE
Add a 4k logo to media

### DISCUSSION THREAD

**Discussion by phaelax** [jellyfin/jellyfin#8309] [General] [answered]
I think it would be a nice subtle feature if 4k media had some kind of small logo overlayed on the artwork.

For example:
![4k_example](https://user-images.githubusercontent.com/23143480/186441333-b72e80de-0cb7-4c8e-a25f-e03b9e686bff.jpg)
.

**Comment by crobibero** [comment, marked as answer]
Feature requests belong at https://features.jellyfin.org

**Reply by phaelax** [reply]
Done.
